addons/binding/org.openhab.binding.miio/README.md

```diff
@@ -0,0 +1,54 @@
+# Xiaomi Robot Vacuum Binding
+
+This Binding is used to control Xiaomi Mi IO devices.
+
+![MIIO logo](doc/miio.png)
+
+## Supported Things
+
+This Binding supports Xiaomi Mi IO devices (Several Xiaomi wifi devices).
+Currently mainly the Xiaomi Robot Vacuum is supported, remaining devices are experimental.
+Experimental means that most likely you can send commands to the devices
+
+The following things are available
+
+| Thing Type | Description |
+|--------|---------| 
+| miio:generic | Generic type for discovered devices. Once the tolken is available and the device model is determined, this thingtype will automatically change to the appropriate thingtype for the model |
+| miio:vacuum | For Xiaomi Robot Vacuum |
+| miio:basic | For several basic devices like yeelights, airpurifiers. Channels and commands are determined by database configuration |
+| miio:unsupported | For experimenting with other devices which use the Mi IO protocol)
```

Comment:
Please structure the table, http://markdowntable.com
Ending `|` is missing in the last line.